server/test/http_tests.js

```diff
@@ -61,7 +61,7 @@ const all_tests = () => {
       });
 
       it('localhost/horizon/horizon.js', (done) => {
-        require(transport).get({ host: http_server.address().address,
+        require(transport).get({ host: "localhost",
```

Comment:
It looks like `address` isn't resolving properly on CircleCI. I think it should be ok to always use `localhost` here.